README.md

```diff
@@ -70,21 +70,30 @@ Then open your browser and visit http://cloud.paddlepaddle.org.
 ### Run locally
 Make sure you are using a virtual environment of some sort (e.g. `virtualenv` or
 `pyenv`).
+
 ```
 virtualenv paddlecloudenv
 # enable the virtualenv
 source paddlecloudenv/bin/activate
 ```
 
 To run for the first time, you need to:
+
 ```
+cd paddlecloud
 npm install
 pip install -r requirements.txt
 ./manage.py migrate
 ./manage.py loaddata sites
 npm run dev
 ```
 
+If `npm` haven't been installed, you need to 
```

Comment:
The command may differ for different OSs, just add one line in Pre-Requirements: "nodejs"